drivers/metal/rendering_context_driver_metal.mm

```diff
@@ -137,6 +165,30 @@ Error resize(uint32_t p_desired_framebuffer_count) override final {
 			frame_buffers[i].set_texture_count(1);
 		}
 
+		if (hdr_output) {
+			layer.wantsExtendedDynamicRangeContent = YES;
+
+			if (hdr_output_prefer_high_precision) {
+				layer.colorspace = CGColorSpaceCreateWithName(kCGColorSpaceExtendedLinearDisplayP3);
```

Comment:
This looks like a memory leak (unless `colorspace` doesn't retain the value passed to it).

Reply:
Ahh, good point!

…and 👋🏻 hey @MaddTheSane!